### ISSUE TITLE
Error when installing using UnityPackageManager 

### ISSUE THREAD

**Issue by cdr9042** [mob-sakai/SoftMaskForUGUI#71]
**Describe the bug**
I got these errors after pasting the line ""com.coffee.softmask-for-ugui": "https://github.com/mob-sakai/SoftMaskForUGUI.git"," into my manifest.json

> [Package Manager] Done registering packages in 0.73s seconds
> Asset Packages/com.coffee.softmask-for-ugui/Assets has no meta file, but it's in an immutable folder. The asset will be ignored. 
> (Filename: C:\buildslave\unity\build\Modules/AssetDatabase/Editor/V2/GuidDB.cpp Line: 614)
> 
> Asset Packages/com.coffee.softmask-for-ugui/CHANGELOG.md has no meta file, but it's in an immutable folder. The asset will be ignored. 
> (Filename: C:\buildslave\unity\build\Modules/AssetDatabase/Editor/V2/GuidDB.cpp Line: 614)
> 
> Asset Packages/com.coffee.softmask-for-ugui/LICENSE.md has no meta file, but it's in an immutable folder. The asset will be ignored. 
> (Filename: C:\buildslave\unity\build\Modules/AssetDatabase/Editor/V2/GuidDB.cpp Line: 614)
> 
> Asset Packages/com.coffee.softmask-for-ugui/package.json has no meta file, but it's in an immutable folder. The asset will be ignored. 
> (Filename: C:\buildslave\unity\build\Modules/AssetDatabase/Editor/V2/GuidDB.cpp Line: 614)
> 
> Asset Packages/com.coffee.softmask-for-ugui/ProjectSettings has no meta file, but it's in an immutable folder. The asset will be ignored. 
> (Filename: C:\buildslave\unity\build\Modules/AssetDatabase/Editor/V2/GuidDB.cpp Line: 614)
> 
> Asset Packages/com.coffee.softmask-for-ugui/ProjectSettings/AudioManager.asset has no meta file, but it's in an immutable folder. The asset will be ignored. 
> (Filename: C:\buildslave\unity\build\Modules/AssetDatabase/Editor/V2/GuidDB.cpp Line: 614)
> 
> Asset Packages/com.coffee.softmask-for-ugui/ProjectSettings/ClusterInputManager.asset has no meta file, but it's in an immutable folder. The asset will be ignored. 
> (Filename: C:\buildslave\unity\build\Modules/AssetDatabase/Editor/V2/GuidDB.cpp Line: 614)
> 
> Asset Packages/com.coffee.softmask-for-ugui/ProjectSettings/DynamicsManager.asset has no meta file, but it's in an immutable folder. The asset will be ignored. 
> (Filename: C:\buildslave\unity\build\Modules/AssetDatabase/Editor/V2/GuidDB.cpp Line: 614)
> 
> Asset Packages/com.coffee.softmask-for-ugui/ProjectSettings/EditorBuildSettings.asset has no meta file, but it's in an immutable folder. The asset will be ignored. 
> (Filename: C:\buildslave\unity\build\Modules/AssetDatabase/Editor/V2/GuidDB.cpp Line: 614)
> 
> Asset Packages/com.coffee.softmask-for-ugui/ProjectSettings/EditorSettings.asset has no meta file, but it's in an immutable folder. The asset will be ignored. 
> (Filename: C:\buildslave\unity\build\Modules/AssetDatabase/Editor/V2/GuidDB.cpp Line: 614)
> 
> Asset Packages/com.coffee.softmask-for-ugui/ProjectSettings/GraphicsSettings.asset has no meta file, but it's in an immutable folder. The asset will be ignored. 
> (Filename: C:\buildslave\unity\build\Modules/AssetDatabase/Editor/V2/GuidDB.cpp Line: 614)
> 
> Asset Packages/com.coffee.softmask-for-ugui/ProjectSettings/InputManager.asset has no meta file, but it's in an immutable folder. The asset will be ignored. 
> (Filename: C:\buildslave\unity\build\Modules/AssetDatabase/Editor/V2/GuidDB.cpp Line: 614)
> 
> Asset Packages/com.coffee.softmask-for-ugui/ProjectSettings/NavMeshAreas.asset has no meta file, but it's in an immutable folder. The asset will be ignored. 
> (Filename: C:\buildslave\unity\build\Modules/AssetDatabase/Editor/V2/GuidDB.cpp Line: 614)
> 
> Asset Packages/com.coffee.softmask-for-ugui/ProjectSettings/NetworkManager.asset has no meta file, but it's in an immutable folder. The asset will be ignored. 
> (Filename: C:\buildslave\unity\build\Modules/AssetDatabase/Editor/V2/GuidDB.cpp Line: 614)
> 
> Asset Packages/com.coffee.softmask-for-ugui/ProjectSettings/Physics2DSettings.asset has no meta file, but it's in an immutable folder. The asset will be ignored. 
> (Filename: C:\buildslave\unity\build\Modules/AssetDatabase/Editor/V2/GuidDB.cpp Line: 614)
> 
> Asset Packages/com.coffee.softmask-for-ugui/ProjectSettings/ProjectSettings.asset has no meta file, but it's in an immutable folder. The asset will be ignored. 
> (Filename: C:\buildslave\unity\build\Modules/AssetDatabase/Editor/V2/GuidDB.cpp Line: 614)
> 
> Asset Packages/com.coffee.softmask-for-ugui/ProjectSettings/ProjectVersion.txt has no meta file, but it's in an immutable folder. The asset will be ignored. 
> (Filename: C:\buildslave\unity\build\Modules/AssetDatabase/Editor/V2/GuidDB.cpp Line: 614)
> 
> Asset Packages/com.coffee.softmask-for-ugui/ProjectSettings/QualitySettings.asset has no meta file, but it's in an immutable folder. The asset will be ignored. 
> (Filename: C:\buildslave\unity\build\Modules/AssetDatabase/Editor/V2/GuidDB.cpp Line: 614)
> 
> Asset Packages/com.coffee.softmask-for-ugui/ProjectSettings/TagManager.asset has no meta file, but it's in an immutable folder. The asset will be ignored. 
> (Filename: C:\buildslave\unity\build\Modules/AssetDatabase/Editor/V2/GuidDB.cpp Line: 614)
> 
> Asset Packages/com.coffee.softmask-for-ugui/ProjectSettings/TimeManager.asset has no meta file, but it's in an immutable folder. The asset will be ignored. 
> (Filename: C:\buildslave\unity\build\Modules/AssetDatabase/Editor/V2/GuidDB.cpp Line: 614)
> 
> Asset Packages/com.coffee.softmask-for-ugui/ProjectSettings/UnityConnectSettings.asset has no meta file, but it's in an immutable folder. The asset will be ignored. 
> (Filename: C:\buildslave\unity\build\Modules/AssetDatabase/Editor/V2/GuidDB.cpp Line: 614)
> 
> Asset Packages/com.coffee.softmask-for-ugui/README.md has no meta file, but it's in an immutable folder. The asset will be ignored. 
> (Filename: C:\buildslave\unity\build\Modules/AssetDatabase/Editor/V2/GuidDB.cpp Line: 614)
> 
> Asset Packages/com.coffee.softmask-for-ugui/release.sh has no meta file, but it's in an immutable folder. The asset will be ignored. 
> (Filename: C:\buildslave\unity\build\Modules/AssetDatabase/Editor/V2/GuidDB.cpp Line: 614)
> 
> Asset Packages/com.coffee.softmask-for-ugui/UnityPackageManager has no meta file, but it's in an immutable folder. The asset will be ignored. 
> (Filename: C:\buildslave\unity\build\Modules/AssetDatabase/Editor/V2/GuidDB.cpp Line: 614)
> 
> Asset Packages/com.coffee.softmask-for-ugui/UnityPackageManager/manifest.json has no meta file, but it's in an immutable folder. The asset will be ignored. 
> (Filename: C:\buildslave\unity\build\Modules/AssetDatabase/Editor/V2/GuidDB.cpp Line: 614)

**To Reproduce**
Find the manifest.json file in the Packages folder of your project and edit it to look like this:

{
  "dependencies": {
    "com.coffee.softmask-for-ugui": "https://github.com/mob-sakai/SoftMaskForUGUI.git",
    ...
  },
}

**Environment (please complete the following information):**
 - Platform: Editor Window
 - Unity: 2019.3.0f6
 - SoftMaskForUGUI: 0.9.1

**Additional context**
Add any other context, screenshots or gif animations about the bug here.

**Comment by mob-sakai** [comment]
@cdr9042 

Sorry for the inconvenience.
This issue will be fixed in `v1.0.0`.

**Comment by mob-sakai** [comment]
### Alternative solution: use upm branch

```
{
  "dependencies": {
    "com.coffee.softmask-for-ugui": "https://github.com/mob-sakai/SoftMaskForUGUI.git#upm",
    ...
   }
}
```

**Comment by MartinHaeusler** [comment]
I second this. The package is working really well for me but the errors produced by the unity editor are quite annoying. Using the `#upm` git URL resolved the issue for me.

**Comment by mob-sakai** [comment]
I have released `v1.0.0-preview.1` which fixes this bug. :smile:
